services/src/datasets/external/netcdfcf/mod.rs

```diff
@@ -146,31 +149,54 @@ pub struct NetCdfEntity {
 }
 
 trait ToNetCdfSubgroup {
-    fn to_net_cdf_subgroup(&self, compute_stats: bool) -> Result<NetCdfGroup>;
+    fn to_net_cdf_subgroup(
+        &self,
+        group_path: &Path,
+        stats_for_group: &HashMap<String, (f64, f64)>,
```

Comment:
I guess this parametrization requires some documentation. Why is it optional to compute stats anyway?

Reply:
It is optional because we don't do this when we load them without preprocessing. It would just take too much time.

Reply:
[d06d470](https://github.com/geo-engine/geoengine/pull/605/commits/d06d470a9e33e9c9b232a593549cd724a76c2550)

Reply:
I still don't get it. Do I have to pass ("group_name", (0, 0))" in here if I want the stats to be computed?

Reply:
You can't compute stats here anymore, because we don't generate them on the NetCDF anymore. You have to pass-in stats.